src/doc/cli_manual/functions.rst

```diff
@@ -1077,17 +1053,17 @@ CopyAnnotationsToWindow
 
   CopyAnnotationsToWindow(source, dest) -> integer
   CopyLightingToWindow(source, dest) -> integer
-  CopyViewTowindow(source, dest) -> integer
+  CopyViewToWindow(source, dest) -> integer
   CopyPlotsToWindow(source, dest) -> integer
 
 
-source : type
+source : integer
     The index (an integer from 1 to 16) of the source window.
 
-dest : type
+dest : integer
     The index (an integer from 1 to 16) of the destination window.
 
-return value : integer
+return type : integer
```

Comment:
Thinking out loud here and am just wondering...we could have verbiage somewhere near the top of the document that says all `integer` return types mean `0 (zero) on failure, non-zero on success`. Or, maybe define a placeholder `CLI_return_t` type (or some such conceptual tool) that gets separately defined with that verbiage.

Reply:
Is the intention here is to explicitly indicate that these integer outputs are just used to indicate failure or success? If so, I like this idea. Is this the case for all the functions that return an integer? I wonder if there is a way to update the script to detect a difference. Otherwise, it may need to be done manually.

Reply:
Well, indeed there appear to be many instances of the phrase, more or less, `returns 1 on success, 0 on failure`. So, I think this is an overriding design feature of the CLI and that maybe separately documenthing this in one paragraph, *somewhere* and then just linking to it for any function whos return behavior indicates that is a little better.